### ISSUE TITLE
Detection of duplicate URLs when submitting a CFP does not seem to work anymore

### ISSUE THREAD

**Issue by jfversluis** [jfversluis/CfpExchange#67]
Entering a URL that is already in the system triggered a warning, prompting the user to double-check. For some reason that is not working anymore.

**Comment by rickvdbosch** [comment]
I've tested this locally: it works fine there. When firing the same request to the production environment, I get HttpStatus 500. My guess is there's a discrepancy between the code and the database. However, I cannot test against the database since I don't have one...

**Comment by jfversluis** [comment]
Hm that is weird. But the 500 might be some kind of CORS thing? So not everyone can just go around and see what is in our database? Just guessing here, I should check App Insights for that.

**Comment by Jandev** [comment]
@rickvdbosch I have a test database set up in Azure which you can use to test this, if you want.

But I figure you can create one for yourself also ;-)

**Comment by rickvdbosch** [comment]
> But the 500 might be some kind of CORS thing?  

Nope, the 500 is the same as the message showing up in Developer Tools in the browser. And calling from Postman isn't seen as cross origin because it's not from another front-end. So there's something not completely right there. I'll be setting up a database to test this, will keep you posted.
